docs/semgrep-code/policies.md

```diff
@@ -24,30 +24,6 @@ Object.entries(frontMatter).filter(
 
 The Policies page displays a visual representation of the rules that Semgrep Code uses for scanning. By default, the same rules will run on all repositories. This is sufficient for most users and simpler for most teams to reason about.
 
-## Multiple policies
```

Comment:
I moved this content down on the page to make it less prominent, and changed the box to let folks know that the beta is not currently open. That way it's still available for folks who have the feature but isn't right in front when it's not the primary experience.

Reply:
ACK